### PR TITLE
fix for the input pickers

### PR DESCRIPTION
**Pull request by yyanwang**
**What is the problem this PR is trying to solve?**

**What is the chosen solution to this problem?**

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js [hunk 1]
```diff
@@ -67,14 +68,13 @@ export default function InputDatePicker(props) {
 	return (
 		<Date.Manager
 			value={props.value}
-			textInput={props.textInput}
```
Comment (yyanwang) [inline packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js]: we don't need textInput passed, because value of date input is only determined by `props.value`

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
-	background: $tc-datetimepicker-background;
-	box-shadow: $tc-datetimepicker-box-shadow;
-	z-index: $tc-datetimepicker-z-index;
+	width: $tc-datepicker-width;
+	background: $tc-datepicker-background;
+	box-shadow: $tc-datepicker-box-shadow;
+	z-index: $tc-datepicker-z-index;
 }
 .date-picker {
+	width: fit-content;
```
Comment (yyanwang) [inline packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss]: This is to make wrapper(FocusManager) of date input shrink, so click outside of input won't open picker.

File: packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js [hunk 3]
```diff
@@ -30,15 +30,16 @@ function InputDateTimePicker(props) {
...
 						<InputDatePicker
 							id={`${props.id}-date-picker`}
+							readOnly={props.readOnly}
+							onBlur={props.onBlur}
 							onChange={onDateChange}
 							value={date}
 							dateFormat={props.dateFormat}
-							useUTC={props.useUTC}
-							timezone={props.timezone}
-							hideTimezone
```
Comment (yyanwang) [inline packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js]: we don't need to pass utc/timezone related props to InputDatePicker anymore, because utc/timezone will be handled on datetime level.

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js [hunk 4]
```diff
@@ -90,13 +90,18 @@ export default function InputDatePicker(props) {
 }
 InputDatePicker.displayName = 'InputDatePicker';
 
+InputDatePicker.defaultProps = {
+	dateFormat: 'YYYY-MM-DD',
+	required: true,
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.component.js]: Why is it required by default ?

File: packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss (hunk 2 above)
Comment (jsomsanith-tlnd) [reply packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss]: Sorry I don't understand, can you rephrase ?

Comment (yyanwang) [reply packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss]: https://www.loom.com/share/f8a2ef679b234e409a15bb756f2af4d1
I made a screen recording, if I don't set width, click the whole line will open the picker. I want to only open picker when click on input field.
